sympy/printing/octave.py

```diff
@@ -432,6 +435,26 @@ def _print_LambertW(self, expr):
         return "lambertw(" + args + ")"
 
 
+    def _print_math_func(self, expr, nest=False):
```

Comment:
When should `nest=False` be used?

Reply:
I don't think it has any specific use here. Should I remove it ?
@jksuom

Reply:
I wonder where that did come from. Was there a similar method somewhere?

Reply:
Yes... It was present in ccode.py..

Reply:
That is in `C99CodePrinter` but not in `C89CodePrinter`. I don't know which one would be more suitable for octave.

Reply:
its needed in ccode because Min(x, y, z) -> fmin(x, fmin(y, z)) whereas fma(x, y, z) -> fma(x, y, z).
so unless there are 3 argument functions in octave I don't think it's needed.

Reply:
I think it's better to remove the nest part and keep the code under the if block.

@bjodah, could you be more specific about the test case you wanted. Were you talking of the ccode(Max(x, x*x).... involving "Max" and "Pow" ?

Reply:
Yes, try to do without `nest`. I'll described the test case some more below.